lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.spec.ts

```diff
@@ -661,48 +643,71 @@ describe('AttachFileCloudWidgetComponent', () => {
             expect(contentModelFormFileHandlerSpy).not.toHaveBeenCalled();
         });
 
-        it('should have been called onInit when widget only one file', async () => {
+        it('should be called onInit when widget has only one file', async () => {
```

Comment:
it's still incorrect in terms of English, the proper might be "should call onInit when widget has only one file"... second concern, this test should not be testing onInit at all...

Reply:
the above is under `describe('contentModelFormFileHandler')`, so I think the form here is good:
`it('should be called onInit when widget has only one file'` - `it` is `contentModelFormFileHandler`.

About testing onInit - Is there a way to do it properly?

Reply:
'should be called OnInit' is not the correct form of English

Reply:
Changed to `'should be called during initialization...'`